aikau/src/main/resources/alfresco/lists/views/layouts/Grid.js

```diff
@@ -394,7 +421,7 @@ define(["dojo/_base/declare",
          {
             focusedChild.blur();
          }
-         domClass.remove(focusedChild.domNode.parentNode, "alfresco-lists-views-layouts-Grid__cell--focused");
+         (focusedChild.domNode && focusedChild.domNode.parentNode) && domClass.remove(focusedChild.domNode.parentNode, "alfresco-lists-views-layouts-Grid__cell--focused");
```

Comment:
Whilst technically true, it helps when reviewing the code... I didn't put it in `if` block just for you, you know ! :)